### PR TITLE
[16.0][MIG] sale_order_product_recommendation

### PR DESCRIPTION
**Pull request by SoporteOperu**

### CODE REVIEW

File: sale_order_product_recommendation/wizards/sale_order_recommendation_view.xml [hunk 1]
```diff
@@ -18,7 +18,12 @@
                         <field name="use_delivery_address" />
                     </group>
                     <group>
```
Comment (Yadier-Tecnativa) [inline sale_order_product_recommendation/wizards/sale_order_recommendation_view.xml:20]: Maybe you should add col="2" to this group

File: sale_order_product_recommendation/tests/test_recommendation.py [hunk 2]
```diff
@@ -25,7 +25,6 @@ def test_recommendations(self):
                 },
             )
         ]
-        self.new_so.order_line.product_id_change()
```
Comment (Yadier-Tecnativa) [inline sale_order_product_recommendation/tests/test_recommendation.py]: This you can change with: self.new_so.order_line._onchange_product_id_warning() the new way to call the method

**Review by Yadier-Tecnativa — CHANGES_REQUESTED** [review]

**Comment by rafaelbn** [comment]
Hello  @SoporteOperu , thank you for this migration! 😄 

Are you going to continue with it?

Ping @sergio-teruel @Juangomezzurita

**Comment by rafaelbn** [comment]
Sorry this modules is already migrated in 

https://github.com/OCA/sale-workflow/tree/16.0/sale_order_product_recommendation

I was lookging for sale_order_product_recommendation_product_sold_by_delivery_week

Thanks

**Pull request closed without merging**